.gitignore

```diff
@@ -3,12 +3,19 @@
 .bundle
 build/*
 tools/chef/cookbooks/*
+tools/chef/site-cookbooks/**/cookbooks/*
 tools/chef/tmp/*
 tools/assets/*
 bin/*
 .DS_Store
 secret.key
 
+# Chef
+**/.kitchen/
+**/cucumber/Gemfile.lock
+tools/chef/site-cookbooks/*/Berksfile.lock
+tools/chef/.chef/*_encrypted_data_bag_secret
```

Comment:
there's usually no need for separate keys between pipeline and production, considering the same two sets of people that do it are the same